setup/fio-setup-basic

```diff
@@ -175,6 +175,12 @@ __numa_cpu_nodes()
 	esac
 }
 
+is_mount_on_root()
+{
```

Comment:
can you update commit subject to be something like below to reflect the relation with root?

fio: run test on /lkp/benchmarks/fio when storage being mounted on root